sentry-android-fragment/src/main/java/io/sentry/android/fragment/SentryFragmentLifecycleCallbacks.kt

```diff
@@ -55,6 +74,8 @@ class SentryFragmentLifecycleCallbacks(
 
     override fun onFragmentResumed(fragmentManager: FragmentManager, fragment: Fragment) {
         addBreadcrumb(fragment, "resumed")
+
+        stopTracing(fragment)
```

Comment:
These always run in the UI thread. rigth? So we're good with `mutable map` and no synchronization here?

Reply:
yes, same with `ActivityLifecycleIntegration`, no need for synchronization or thread-safe map.